utoipa-gen/tests/request_body_derive_test.rs

```diff
@@ -432,6 +432,42 @@ fn derive_request_body_ref_path_success() {
     assert_eq!(component_ref, "#/components/schemas/path.to.Foo");
 }
 
+#[test]
+fn unit_type_request_body() {
+    type UT = ();
+
+    #[utoipa::path(
+        post,
+        path = "/unit_type_test",
+        request_body = UT
```

Comment:
Just wondering whether you are able now to directly use the `request_body = ()` because the change you made to `media_type_schema.rs` file for the tuple

Reply:
I just checked and it's not possible. I get the following error message: "ValueType::Primitive must have path".

Reply:
Okay. Hm.. then there is something else missing as well.. Though I can check this later as well. It should not be primitive type but `ValueType::Tuple`